Makefile

```diff
@@ -24,7 +24,14 @@ EMBEDDED_OPERATOR_UTILS_IMAGE_VERSION ?= $(subst +,-,$(VERSION))
 EMBEDDED_OPERATOR_UTILS_IMAGE_LOCATION = proxy.replicated.com/anonymous/$(EMBEDDED_OPERATOR_UTILS_IMAGE):$(EMBEDDED_OPERATOR_UTILS_IMAGE_VERSION)
 EMBEDDED_CLUSTER_OPERATOR_IMAGE_OVERRIDE =
 OPENEBS_CHART_VERSION = 4.1.0
-OPENEBS_UTILS_VERSION = 4.1.0
+OPENEBS_IMAGE_VERSION = 4.1.0
+OPENEBS_KUBECTL_IMAGE_VERSION = 1.29
```

Comment:
I think i still have some work here to figure out exactly what version will end up as the input here. the default version is 1.25.15 but this is not available in chainguard.

https://github.com/openebs/openebs/blob/main/charts/values.yaml#L58

Reply:
What about using the latest for the K0S major version ?